app/controllers/hikes_controller.rb

```diff
@@ -52,7 +52,10 @@ def show
     authorize @booking
     related_difficulty = @hike.difficulty
     related_region = @hike.region
-    @related_hikes = Hike.where(difficulty: related_difficulty).or(Hike.where(region: related_region)).sample(4).&(Hike.where.not(id:@hike.id))
+    related_accomodation = @hike.accomodation_type
+
+
+@related_hikes = Hike.where(region: related_region).or(Hike.where(difficulty: related_difficulty)).or(Hike.where(accomodation_type: related_accomodation))&.where.not(id: @hike.id).sample(4)
```

Comment:
Please add indentation (4 spaces/tab) for consistency on code style.

The query looks good 👍